internal/pkg/deploy/cloudformation/workload.go

```diff
@@ -35,7 +35,7 @@ func (cf CloudFormation) DeployService(conf StackConfiguration, opts ...cloudfor
 	return cf.cfnClient.UpdateAndWait(stack)
 }
 
-// DeleteService removes the CloudFormation stack of a deployed service.
-func (cf CloudFormation) DeleteService(in deploy.DeleteWorkloadInput) error {
+// DeleteWorkload removes the CloudFormation stack of a deployed workload.
+func (cf CloudFormation) DeleteWorkload(in deploy.DeleteWorkloadInput) error {
 	return cf.cfnClient.DeleteAndWait(fmt.Sprintf("%s-%s-%s", in.AppName, in.EnvName, in.Name))
```

Comment:
Can we make this `DeleteWorkload` to be a private method as `deleteWorkload` and having two more `DeleteService` and `DeleteJob`, so that we could wrap the error and make the error message more helpful and specifically for service/job. Thank you 🙏🏿

Reply:
Oh, I wrapped the errors where `DeleteWorkload` is called, in `svc_delete.go` and `job_delete.go`. Is that okay? Or is it preferable to wrap them higher up in the chain like you suggest here?